hack/verify-readonly-packages.sh

```diff
@@ -45,7 +45,7 @@ find_files() {
 IFS=$'\n'
 conflicts=($(find_files | sed 's|/.readonly||' | while read dir; do
     dir=${dir#./}
-    if kube::util::has_changes_against_upstream_branch "${branch}" "^${dir}/[^/]*\$" "/.readonly\$|/BUILD$|/zz_generated" &>/dev/null; then
+    if kube::util::has_changes_against_upstream_branch "${branch}" "^${dir}/[^/]*\$" '/\.readonly$|/BUILD$|/zz_generated|/\.generated\.|\.proto$|\.pb\.go$' >/dev/null; then
```

Comment:
and the other is generated.pb.go?

I might have listed the files explicitly, but I don't feel strongly.